projects/objectnav_baselines/models/object_nav_models.py

```diff
@@ -44,54 +38,84 @@ def __init__(
         action_space: gym.spaces.Discrete,
         observation_space: SpaceDict,
         goal_sensor_uuid: str,
-        rgb_uuid: Optional[str],
-        depth_uuid: Optional[str],
+        # RNN
         hidden_size=512,
-        object_type_embedding_dim=8,
-        trainable_masked_hidden_state: bool = False,
         num_rnn_layers=1,
         rnn_type="GRU",
+        add_prev_actions=False,
+        action_embed_size=6,
+        # Aux loss
+        multiple_beliefs=False,
+        beliefs_fusion: Optional[str] = None,
+        auxiliary_uuids: Optional[List[str]] = None,
+        # below are custom params
+        rgb_uuid: Optional[str] = None,
+        depth_uuid: Optional[str] = None,
+        object_type_embedding_dim=32,
```

Comment:
Can you keep the default `object_type_embedding_dim=8` from the previous model definition?